packages/itinerary-body/src/TransitLegBody/index.js

```diff
@@ -39,6 +39,37 @@ DefaultTransitLegSummary.propTypes = {
   stopsExpanded: PropTypes.bool.isRequired
 };
 
+function DefaultRouteDescription({ leg }) {
+  const { headsign, routeLongName, routeShortName } = leg;
+  return (
+    <Styled.LegDescriptionForTransit>
+      {routeShortName && (
+        <div>
+          <Styled.LegDescriptionRouteShortName>
+            {routeShortName}
+          </Styled.LegDescriptionRouteShortName>
+        </div>
+      )}
+      <Styled.LegDescriptionRouteLongName>
+        {routeLongName}
+        {headsign && (
+          <span>
+            {" "}
```

Comment:
Roll these spaces into the CSS for `<Styled.LegDescriptionHeadsignPrefix>`, either as :before/:after content or as margin/padding? Then remove the surrounding `<span>` if it doesn't accomplish anything that `<Styled.LegDescriptionHeadsignPrefix>` can't already do.

Reply:
That sounds a little too fancy. This approach results in the content being all in one place which seems a little simpler and easier to maintain.

Reply:
Another way to do it is to roll the spaces like this: `<Styled.LegDescriptionHeadsignPrefix>{" to "}</Styled.LegDescriptionHeadsignPrefix>`.

Reply:
That looks the cleanest of all.